amazonsqs/amazonsqs_data_mappings.bal

```diff
@@ -0,0 +1,178 @@
+// Copyright (c) 2019 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/log;
+import ballerina/io;
+
+function jsonToCreatedQueueUrl(json source) returns string {
+    json queueUrl = source.CreateQueueResponse.CreateQueueResult.QueueUrl;
+    return queueUrl != null ? queueUrl.toString() : EMPTY_STRING;
+}
+
+function jsonToOutboundMessage(json source) returns OutboundMessage {
+    json msgSource = source.SendMessageResponse.SendMessageResult != null ? source.SendMessageResponse.SendMessageResult : {};
```

Comment:
Some lines are more than 120 chars. Shall we refractor those?

Reply:
Corrected